batcher/aligned-batcher/src/lib.rs

```diff
@@ -974,20 +1310,34 @@ impl Batcher {
 
         unlock_block != U256::zero()
     }
+
+    async fn get_gas_price(&self) -> Option<U256> {
+        match self.eth_ws_provider.get_gas_price().await {
+            Ok(gas_price) => Some(gas_price), // this is the block's max priority gas price, not the base fee
```

Comment:
considering this equation `maxFeePerGas = baseFeePerGas + maxPriorityFeePerGas`, do you mean that the gas price obtained here is `maxPriorityFeePerGas = maxFeePerGas - baseFeePerGas`?

Reply:
I think @uri-99 means both when we set it, but its actually the maxFeePer gas we obtain according to this tx: https://holesky.etherscan.io/tx/0x3cd9998931b66b2140af28de6437c35d15ed9e310a99aaa2428e08f81ce14a87

where fee for aggregator is 78065194400000 which divided by 400_000 (gas for aggregator) is 195_162_986 (maxFeePerGas / maxPriorityFeePerGas which are the same on the tx)

Reply:
Yes, its maxFeePerGas, not only the priority component